packages/admin/cms-admin/src/contentScope/ContentScopeSelect.tsx

```diff
@@ -107,7 +107,14 @@ export function ContentScopeSelect({
                         <Domain />
                     </ListItemIcon>
                     <ListItemText
-                        primaryTypographyProps={{ variant: "body2", fontWeight: "inherit" }}
+                        slotProps={{
+                            primary: {
+                                variant: "body2",
+                                sx: {
+                                    fontWeight: isSelected ? "600 !important" : 250,
```

Comment:
It seems odd that `!important` is only required for one value but not the other... 
Is the `250` applied correctly from here, or does the `600` maybe not need it?